resources/assets/components/InboxItem/index.js

```diff
@@ -39,6 +41,22 @@ class InboxItem extends React.Component {
     const campaign = this.props.details.campaign;
     const signup = this.props.details.signup;
 
+    if (post['user']) {
```

Comment:
Got it! Nah, let's add a TODO referencing that & if you're feeling up to it we can always tackle in another PR afterwards.